### PR TITLE
Feature/sidebar-component

### PR DESCRIPTION
**Pull request by misspelz**
sidebar

### CODE REVIEW

File: src/components/Quote/Quote.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Quote.propTypes = {
+  content: PropTypes.string,
+  boldcontent: PropTypes.string,
+  className: PropTypes.string,
+  dataTest: PropTypes.string
+};
+
+Quote.defaultProps = {
+  // boldcontent: 'Bold Text',
```
Comment (edustipend-root) [inline src/components/Quote/Quote.jsx:21]: Hello @misspelz, please remove this comment.

File: src/components/SideBar/contants.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    title: 'My Account',
+    icon: User,
+    link: 'account'
+  }
+];
+
+export const quoteData = [
+  {
+    id: 1,
+    boldquote: 'Did you know?',
```
Comment (edustipend-root) [inline src/components/SideBar/contants.js:29]: Can we rename this to `title` or `header` instead of `boldquote`?

**Review by edustipend-root — CHANGES_REQUESTED** [review]
LGTM! Just some minor changes to be made.
Good work @misspelz!

File: src/components/SideBar/contants.js (hunk 2 above)
Comment (misspelz) [reply src/components/SideBar/contants.js]: Done

File: src/components/Quote/Quote.jsx (hunk 1 above)
Comment (misspelz) [reply src/components/Quote/Quote.jsx]: Done

**Review by edustipend-root — APPROVED** [review]
Looks good @misspelz, merging now!

**Pull request merged**